extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/config/VaultRuntimeConfig.java

```diff
@@ -50,6 +51,28 @@
     @ConfigDocSection
     public VaultAuthenticationConfig authentication;
 
+    /**
```

Comment:
I propose to introduce a `Health` configuration group as these 3 parameters really stand up on their own IMHO.
Also, is default value needed for `false` ?

Reply:
Now, correct me if I am not right these properties are under: `quarkus.vault.health.*` isn't it?

Yes by default they are `false`

Reply:
What I have seen in other extensions that they are integrating with the spec, they are following the same approach, but please I am new on extension ecosystem, so I can be totally wrong.

Reply:
You'd have 
```
@ConfigItem
Health health = new Health();
@ConfigGroup
public class Health {
   boolean enabled;
   // etc
}
```

Sure lets keep a default initialization to `false`

Reply:
@lordofthejars It is not a big deal, but it feels it would better be encapsulated in the dedicate group